bedrock/base/templates/includes/structured-data/organization/mozilla-corporation-organisation.json

```diff
@@ -0,0 +1,460 @@
+    "url": "https://www.mozilla.org",
+    "logo": "https://www.mozilla.org/media/img/mozorg/mozilla-256.4720741d4108.jpg",
+    "diversityPolicy": "https://blog.mozilla.org/inclusion/mozilla-workplace-transition-policy-guidelines/",
+    "ethicsPolicy": "https://www.mozilla.org/about/manifesto/",
+    "foundingDate": "2005-08-03",
+    "founder": "Mitchell Baker",
+    "sameAs": [
+      "https://www.wikidata.org/wiki/Q169925",
+      "https://twitter.com/mozilla",
+      "https://www.facebook.com/mozilla",
+      "https://github.com/mozilla",
+      "https://www.instagram.com/mozilla/",
+      "https://www.youtube.com/user/Mozilla",
+      "https://en.wikipedia.org/wiki/Mozilla_Corporation",
+      "https://www.linkedin.com/company/mozilla-corporation"
+    ],
+    "brand": [{
+        "@type": "brand",
+        "@id": "https://www.mozilla.org/#mozilla",
+        "url": "https://www.mozilla.org",
+        "logo": "https://www.mozilla.org/media/img/mozorg/mozilla-256.4720741d4108.jpg",
+        "name": "Mozilla",
+        "sameAs": ["https://en.wikipedia.org/wiki/Mozilla", "https://www.wikidata.org/wiki/Q9661"]
+      },
+      {
+        "@type": "brand",
+        "@id": "https://www.mozilla.org/#firefoxbrand",
+        "url": "https://www.mozilla.org/firefox/",
+        "logo": "https://www.mozilla.org/media/img/logos/firefox/logo-master-wordmark-dark.9969bf7e6192.svg",
+        "name": "Firefox",
+        "sameAs": [
+          "https://twitter.com/firefox",
+          "https://www.instagram.com/firefox/",
+          "https://www.youtube.com/firefox"
+        ]
+      }     ],
+      "owns": [
+      {
+    "@type": "Product",
+    "@id": "https://www.mozilla.org/#firefoxandroid",
+    "url": "https://www.mozilla.org/firefox/mobile/",
+    "logo": "https://www.example.com/example-logo.jpg",
+    "image": [
+        "https://example.com/photos/1x1/photo.jpg",
+        "https://example.com/photos/4x3/photo.jpg",
+        "https://example.com/photos/16x9/photo.jpg"
+        ],
+    "name": "Firefox Browser Android",
+    "alternateName": [ 
+    	"Firefox Android",
+		"Firefox for Android"
+		],
+    "brand": {
+        "@id": "https://www.mozilla.org/#firefoxbrand"
+        },
+    "sameAs": [
+        "https://www.wikidata.org/wiki/Q550315",
+        "https://en.wikipedia.org/wiki/Firefox_for_Android",
+        "https://play.google.com/store/apps/details?id=org.mozilla.firefox"
+        ],
+    "offers": {
+        "@type": "Offer",
+        "url": "https://play.google.com/store/apps/details?id=org.mozilla.firefox",
+        "priceCurrency": "USD",
+        "price": "0",
+        "availability": "https://schema.org/InStock"
+        }
+    	},
+    	{
+    "@type": "Product",
+    "@id": "https://www.mozilla.org/#firefoxbeta",
+    "url": "https://www.mozilla.org/en-US/firefox/channel/desktop/",
+    "logo": "https://www.example.com/example-logo.jpg",
```

Comment:
I'm assuming this is just placeholder, but we should not be trusting third party domains for our images here imho.